recipes/pygtftk/meta.yaml

```diff
@@ -0,0 +1,57 @@
+package:
+  name: pygtftk
+  version: "0.9.7"
+
+source:
+  url: https://github.com/dputhier/pygtftk/archive/0.9.7.tar.gz
+  md5: 6ce8e5d2351cd039c11dee6814dc6b98
+  sha1: 23862a12605a02de001d540c4e2fa5688294d662
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python >=3.6,<3.7
```

Comment:
can you `skip: True` builds instead of this pinning here

Reply:
If I understand well, I need `skip: true # [not py35 and not py36]`, keep `- python` in host but still `- python >=3.5,<3.7` in run. Am I right? 
By the way, py35 is also ok hence the difference with my commit.